### PR TITLE
Access key switching to split screen overview and back to WP list

### PR DESCRIPTION
**Pull request by knight-bubble**
[`* `#17808`  Define access key to switch between WP list and split screen`](http://community.openproject.org/work_packages/17808)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://hirtie-maxim-fix-17808-access-key-switch.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://hirtie-maxim-fix-17808-access-key-switch.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "gulp-protractor": "0.0.11",
     "gulp-replace": "^0.5.3",
     "gulp-ruby-sass": "^0.7.1",
+    "gulp-util": "^3.0.4",
```
Comment (myabc) [inline frontend/package.json:18]: @hirtie-maxim what's this for? if this is a fix for something else, it should be part of another PR.

Comment (knight-bubble) [inline frontend/package.json:18]: @myabc, there was an error on travis regarding to this library. So I've added this package to see if error repeats or not.

Comment (myabc) [inline frontend/package.json:18]: @hirtie-maxim I was seeing that too. If it's not too hard, can you split the commit and add an appropriate message to that effect?

**Comment by floriank** [comment]
@hirtie-maxim - please split up the PR. The dependency should have nothing to do with the issue addressed.
